src/helpers/Constants.ts

```diff
@@ -131,7 +131,11 @@ const constants = {
   },
   /* Dimensions */
   removeDimensionsEventListener: (callback: any) => {
-    Dimensions.removeEventListener('change', callback);
+    if (Dimensions.remove) {
+      Dimensions.remove('change', callback);
+    } else {
+      Dimensions.removeEventListener('change', callback);
+    }
```

Comment:
@tacoelho I approved, but then noticed the build fails. 
It's probably because of TS. 
The best way to see the errors is to run `npm run prepush` 
Anyway, you can use `@ts-expect-error` for now
```suggestion
    // @ts-expect-error
    if (Dimensions.remove) {
      // @ts-expect-error
      Dimensions.remove('change', callback);
    } else {
      Dimensions.removeEventListener('change', callback);
    }
```